### ISSUE TITLE
Elastic 'list' menu items added to content menu on small screens

### ISSUE THREAD

**Issue by johndoh** [roundcube/roundcubemail#7618]
With the Elastic skin in the Mail view on small or phone size screens when you are on the mail list screen if you click the three dots in the top right corner you get "select" and "threads" and "(list) options" this makes sense. If you then open a message "select" and "threads" and "(list) options" are still in the menu at the top right, this does not make sense, there is no message list and so nothing to select or change the thread mode on or sort. Same is true in Contacts, when viewing a contact you still get the "select" option. I think the list options should only be shown on the list screen. I am creating this ticket because, while I will keeping working on it, I don't think its simple to change this behavior so is it worth the effort?

**Comment by alecpl** [comment]
Makes sense. More>Import option probably should also not be there. This is not a big issue, I'm not sure it's worth the effort.
